.distignore

```diff
@@ -1,22 +1,13 @@
 # A set of files you probably don't want in your WordPress.org distribution
-.distignore
-.editorconfig
-.git
-.gitignore
-.travis.yml
-.DS_Store
-Thumbs.db
-bin
-behat.yml
-circle.yml
+.*
+*.xml
+*.yml
 Gruntfile.js
 package.json
-phpunit.xml
-multisite.xml
 phpunit.xml.dist
-phpcs.ruleset.xml
-README.md
-wp-cli.local.yml
-tests
-/vendor/lcache/lcache/tests
-node_modules
```

Comment:
@danielbachhuber I added trailing slashes to indicate directories for clarity. The others are covered by `*.yml` and `*.xml`. See the additions below.